packages/react-native-editor/ios/Podfile.lock

```diff
@@ -306,7 +306,7 @@ PODS:
     - React-Core
   - RNGestureHandler (2.2.0-wp-4):
     - React-Core
-  - RNReanimated (2.2.4-wp-1):
+  - RNReanimated (2.4.1-wp-1):
```

Comment:
Not necessarily a blocker, but I wonder why this change appears in the release process PR.

Perhaps it's something overlooked in [the original PR](https://github.com/WordPress/gutenberg/pull/39430)? Just to verify, whdy @fluiddot?

Reply:
Yes, I'm afraid I overlooked updating the `Podfile.lock` file on the demo app project when applying the changes in https://github.com/WordPress/gutenberg/pull/39430, that's why it shows up here, sorry for that 🙇 .

Reply:
In any case, this shouldn't block the release PR as the change is expected.

Reply:
Not a prob, just making sure this is not surprising for any reason 😸👍